study.md

```diff
@@ -21,18 +21,22 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+In the JSIS article on callbacks, there is an example where the function `successCallback()` is defined twice. How does this work?
 ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
+In what cases would it be desireable to use a function declaration rather than a function expression? Are there any cases where it is clearly more advantageous to use declarations or is it just a matter of how a developer chooses to structure their code?
```

Comment:
In my opinion and from personal experience, I use function declarations since they seem to avoid the hoisting problems that one may encounter when using function expressions.  Some code bases will prefer function expressions in an attempt to adhere to JavaScript's expressive style, so I think that it's a matter of style and structure.  Personally, if I'm working in someone else's code base, then I'll use whatever style they're using.  For personal projects, I'll use function declarations.